torch/_dynamo/variables/lists.py

```diff
@@ -898,6 +899,10 @@ def call_method(
             return result
         elif name == "__len__":
             return ConstantVariable(len(self.items)).add_options(options)
+        elif name == "__contains__":
+            assert len(args) == 1
+            assert not kwargs
+            return _listlike_contains_helper(self.items, args[0], options)
```

Comment:
I don't know enough about set contains :( we should check the cpython implementation, we might need to call `hash` instead of using raw equality...

Reply:
I think in python, ```__eq__``` and ```__hash__``` must agree, equal objects must have equal hashes. The reason of using ```operator.eq``` is to leverage the ```BuiltinVariable``` where supports ```constants/tensors/modules/symbols``` well.

Reply:
Sounds good :)